chart/templates/content-service-gcloud-secret.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+{{- if .Values.components.contentService.remoteStorage.gcloud }}
+{{- if .Values.components.contentService.remoteStorage.gcloud.credentialsFileUpload }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.components.contentService.remoteStorage.gcloud.secretName | default "ws-sync-gcloud" }}
+  labels:
+    app: {{ template "gitpod.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  "{{ base .Values.components.contentService.remoteStorage.gcloud.credentialsFile }}": {{ $.Files.Get (print (.Values.components.contentService.remoteStorage.gcloud.credentialsFileUploadDir | default "secrets/decrypted/") (base .Values.components.contentService.remoteStorage.gcloud.credentialsFile)) | b64enc | quote }}
+{{- end }}
+{{- end }}
```

Comment:
If we create the secret in helm, creating it can be removed from Terraform:
https://github.com/gitpod-io/gitpod/blob/main/install/gcp-terraform/modules/storage/main.tf#L47

Reply:
I don't think we should add this template. For most other secrets we don't have it either.
Instead, you could just set the `secretName` in the `values.yaml` and copy a ready-made secret from the `werft` namespace.

Reply:
@csweichel: I wanted to make it in sync to the implementation in `gitpod-com`. But yes, copying the secret makes it much easier. With this, we can also drop the `gitpod-com` PR and just leave everything there as it is.

I have updated the PR.